### PR TITLE
Convert repo to computer setup only

### PR DESCRIPTION
**Pull request by bradsmialek**
Converting code-201-prework to computer setup only

### CODE REVIEW

**Comment by ncarignan** [comment]
lets merge this next week @ncarignan, we want to avoid changing the prework while Amanda's class is in the middle of it

File: README.md [hunk 1]
```diff
@@ -1,139 +1,74 @@
...
+	editor = code --wait
+[user]
+	name = yourgithubusername
+	email = youremail
+username@user $ code
+//should open VSCode
+```
+---
+#### Code 102 *(day or night classes)*
+If you think you could benefit from a classroom setting with guidance, consider completing <a href="https://www.codefellows.org/courses/code-102/intro-to-software-development/">Code 102</a>. You’ll receive 30 hours of instruction as you blast through the prework for Code 201, while getting a tour of the tools and techniques of modern developers..
```
Comment (ncarignan) [inline README.md]: if we are requiring 102, let's rephrase this

File: README.md [hunk 2]
```diff
@@ -66,9 +68,10 @@ username@user $ code
...
 ```
 ---
-#### Code 102 *(day or night classes)*
-If you think you could benefit from a classroom setting with guidance, consider completing <a href="https://www.codefellows.org/courses/code-102/intro-to-software-development/">Code 102</a>. You’ll receive 30 hours of instruction as you blast through the prework for Code 201, while getting a tour of the tools and techniques of modern developers..
 
 #### Need help or have questions about computer setup?
+
 Please contact our Admissions Support Instructor: <brad.smialek@codefellows.com> 
 
+If Code 102 self-study isn't right for you, we understand. Ask about Code 102 (day or night classes) in a classroom setting with guidance. You’ll receive 30 hours of instruction as you blast through the prework for Code 201, while getting a tour of the tools and techniques of modern developers..
```
Comment (bradsmialek) [inline README.md]: Rephrased #94

Comment (bradsmialek) [inline README.md]: Rephrased

**Comment by ncarignan** [comment]
code 201 is fine with this

**Comment by brookr** [comment]
Superseded by the stand-alone setup-guide repo now.

**Pull request closed without merging**